.github/workflows/build-image.yml

```diff
@@ -56,15 +56,19 @@ jobs:
           SHA: ${{ github.sha }}
         run: |
           export NODE_VERSION=`cat .nvmrc`
-          export GITHASH="${SHA::9}"
+          export GITHASH="${SHA::7}"
+          export GITHASH_9="${SHA::9}"
           export IMAGE_TAG="githash-${GITHASH}"
+          export IMAGE_TAG_9="githash-${GITHASH_9}"
 
           echo "## Git Hash" >> $GITHUB_STEP_SUMMARY
```

Comment:
We have a step like this in many projects that might be good later in this file (after the metadata step):

```
      - name: Report
        shell: bash
        run: |
          echo "### Tags" >> $GITHUB_STEP_SUMMARY
          echo "${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
          echo '' >> $GITHUB_STEP_SUMMARY
```